### DISCUSSION TITLE
How could this unix socket problem be solved?

### DISCUSSION THREAD

**Discussion by Suchun-sv** [coder/code-server#6550] [Q&A]
I simply download the binary file and run the ~/.local/bin/code-server ., sometime, it can work properly, however, today I find I could not run the service, do anyone help me?
<img width="566" alt="image" src="https://github.com/coder/code-server/assets/35399433/2ba0427c-8915-4e51-a2b2-63e0c312789e">

**Comment by code-asher** [comment]
I think this means code-server is trying to open the current directory in a running instance of code-server, but there is no running instance.

How are you running it?  From inside the integrated terminal of another code-server instance or from an external terminal?  Please post the exact reproduction steps, the expected behavior, and the version of code-server.  Thanks!
